tests/accessor/accessor_api_image.cpp

```diff
@@ -38,19 +38,26 @@ class TEST_NAME : public util::test_base {
     try {
       auto queue = util::get_cts_object::queue();
 
-      /** check image accessor api for cl_int4
-       */
-      check_image_accessor_api_type<cl::sycl::cl_int4>(log, queue);
+      try {
```

Comment:
This check looks correct, however this relies on the implementation throwing a `feature_not_supported` exception, which I believe would be considered negative testing and for the Khronos CTS we can only use positive testing.

We could alter this so that the check for image support is done before doing the checks.

Reply:
I am not sure it is really negative testing:
- if the feature is supported fine;
- if the feature is not supported, it should throw and the check is skipped from the CTS. So it is not really negative testing since it has no side effect;
- if the feature is not supported but does as if, the CTS will probably fail as the operation will not work correctly.

Reply:
I agree with @keryell, that it not really a negative test, but we might want to apply @AerialMantis suggestion for the sake of consistency.
I think the approach we use to skip tests relying on unsupported features should be consistent for the whole project.
Does it make sense?